network.go

```diff
@@ -449,6 +461,13 @@ func (n *network) UnmarshalJSON(b []byte) (err error) {
 	}
 	n.name = netMap["name"].(string)
 	n.id = netMap["id"].(string)
+	// "created" is not available in older versions
+	if v, ok := netMap["created"]; ok {
+		// n.created is time.Time but marshalled as string
+		if err = n.created.UnmarshalText([]byte(v.(string))); err != nil {
+			return err
```

Comment:
Though this is not a concern, I cant think of a reason to fail the Unmarshal if `created` fails. This is not a critical variable of the network object and must be treated as such.